package.json

```diff
@@ -37,6 +37,8 @@
   },
   "homepage": "https://github.com/ForgeRock/forgerock-javascript-sdk#readme",
   "devDependencies": {
+    "@changesets/changelog-github": "^0.5.0",
+    "@changesets/cli": "^2.27.9",
```

Comment:
Saw you removed browserstack and nx-cloud. Should we update this to version not at risk?